.github/workflows/pr_jira_check.yml

```diff
@@ -0,0 +1,24 @@
+name: PR checker and updater for Jira missing stuff
+
+on:
+  pull_request:
+    types: ['opened', 'edited', 'reopened', 'synchronize']
+
+jobs:
+  title:
+    name: Missing Jira ticket check
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Check for ticket
+        uses: neofinancial/ticket-check-action@v1
```

Comment:
I see warnings about that we try to use deprecated versions of node.js (12). Looks like the latest version of this is v2. Can you try to use v2 instead?

Reply:
Let me try on private first and I let you know.

Reply:
@sm6srw I did it on private and it works. PR updated.

Reply:
It fails again, but I don't catch why :( maybe is the name of the branch, or idk 
![image](https://github.com/DynamoDS/Dynamo/assets/168037709/6b914c11-2af7-403f-99b9-6c84e5b48fba)